pkg/controller/controller.go

```diff
@@ -547,13 +548,21 @@ func startServices() error {
 	if err != nil {
 		return fmt.Errorf("Failed to create grpc gateway, %v", err)
 	}
-	mux.Handle("/", gw)
+	mux.Handle("/edgeproto/v1/", gw)
```

Comment:
Did you just want to version our api?

Reply:
That and it needs to be on a different path than the NBI hander.